handlers.go

```diff
@@ -424,9 +424,20 @@ func handleLive(w http.ResponseWriter, r *http.Request) {
 		muxer := flv.NewMuxerWriteFlusher(writeFlusher{httpflusher: flusher, Writer: w})
 		cursor := ch.que.Latest()
 
+		ip := r.Host
+		f := r.Header.Get("Forwarded")
+		xff := r.Header.Get("X-Forwarded-For")
+		if xff == "" {
```

Comment:
This doesn't look right.  If empty, overwrite value with nothing?

Reply:
You're right. 

As the title say, it's still work in progress. 

For me it's 3:45am, so my brain might had left me alone.